dcli/generatorCli.py

```diff
@@ -89,7 +91,7 @@ def java(self):
         self.add_type_if_necessary(self.parse_target_method_args(answers['targetMethodReturn']), language)
 
         # append default value
-        answers[TARGET_METHOD_RETURN_VALUE] = language.get_default_value(answers['targetMethodReturn'])
+        answers[TARGET_METHOD_RETURN_VALUE] = language.get_default_value(answers['targetMethodReturn']) if answers['targetMethodReturn'] else ""
```

Comment:
Why `if answers['targetMethodReturn'] else ""` ? It's already the case, check `get_default_value` in `language.py`.

Jump line next time.

Reply:
Same thing for Kotlin

Reply:
done